reactor-core/src/test/java/reactor/core/scheduler/AbstractSchedulerTest.java

```diff
@@ -103,12 +109,16 @@ protected boolean isTerminated(Scheduler s) {
 	}
 
 	@Test
-	void cannotInitTwice() {
-		Assumptions.assumeThat(shouldCheckInit())
-		           .as("scheduler supports restart prevention").isTrue();
+	@org.junit.jupiter.api.Disabled("Should be enabled in 3.5.0")
```

Comment:
nit: import the class and leave only `@Disabled`

Reply:
if disabled only for a single scheduler, perhaps use a "configuration" methods like `shouldCheckInit()` ?

Reply:
🤦 I was looking for the `@Ignore` counterpart and forgot to import ;)
I'd leave it disabled instead of narrowing down, as that is something that highlights a behaviour change and can be uniformly added in 3.5.0.